src/main/scala/ch/epfl/bluebrain/nexus/kg/routes/ResourceRoutes.scala

```diff
@@ -42,10 +48,12 @@ import scala.concurrent.Future
 import scala.util.Try
 
 class ResourceRoutes(implicit repo: Repo[Task],
-                     projects: Projects[Task],
                      indexers: Clients[Task],
                      store: AttachmentStore[Task, AkkaIn, AkkaOut],
-                     config: AppConfig) {
+                     config: AppConfig,
+                     httpClient: UntypedHttpClient[Task],
```

Comment:
the `httpClient` and `mt` I guess are needed in order to generate the `HttpClient`.
I would put them inside `Clients` case class in the implicit part